packages/npm-packages/.npm/package/npm-shrinkwrap.json

```diff
@@ -80,6 +80,9 @@
     },
     "moniker": {
       "version": "0.1.2"
+    },
+    "underscore": {
+      "version": "1.8.3"
```

Comment:
Actually underscore should be available already. It's a standard Meteor package, and if it wasn't available already it can be added with `meteor add underscore`.